.evergreen/start-orchestration.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+if [ -z "$BASHPID" ]; then
+  echo "start-orchestration.sh must be run in a Bash shell!" 1>&2
```

Comment:
Is this possible given the `#!/usr/bin/env bash` above?

Reply:
Yes. Shebangs only apply if the script is executed without explicitly specifying the shell to use, i.e. `chmod +rx ./script.sh && ./script.sh`. Explicitly using `sh` as in `sh -c script.sh` does not respect the shebang (runs in `sh`, which is often _not_ the same as Bash).

Reply:
Indeed. That tripped me up a few times while implementing downstream tickets for [DRIVERS-2497](https://jira.mongodb.org/browse/DRIVERS-2497) as our Evergreen config was still using `sh` to invoke the scripts that were recently changed to use these shebangs. Easily missed especially when there are a bunch of environment variables before the script path!